api/services/registration_service.py

```diff
@@ -17,18 +17,16 @@ def set_init_data(self, jsonData):
     def handle_registration(self, jsonData):
         try:
             self.set_init_data(jsonData)
-            self.register(jsonData)
+            flow_run_uuid = helpers.fetch_by_key('run_uuid', jsonData)
+            call_log = models.CallLog.query.get_by_flow_run_uuid(flow_run_uuid)
```

Comment:
Seems like duplicate of line after the `if` condition. Please remove

Reply:
yea! removed duplicacy